scripts/prepare_cluster.sh

```diff
@@ -7,14 +7,16 @@ source $VARIABLES
 if [ -z $MASTER_CERTIFICATE_KEY ]
 then
     echo "[info] MASTER_CERTIFICATE_KEY not set. Generating new."
-    MASTER_CERTIFICATE_KEY=$(docker run kubeadocker alpha certs certificate-key)
+    # https://github.com/kubernetes/kubernetes/blob/cde45fb161c5a4bfa7cfe45dfd814f6cc95433f7/cmd/kubeadm/app/phases/copycerts/copycerts.go#L80-L87
+    MASTER_CERTIFICATE_KEY=$(head -c32 /dev/random | sha256sum | awk '{print $1}')
```

Comment:
Why exactly do you compute the `sha256sum`?

Reply:
We need a string, `/dev/random` provides bytes.

Reply:
So it is just a convenient way of turning the bytes into a hex encoded string.
Now why do we care to specify picking 32 bytes from random, the sha256sum always outputs 32 bytes regardless of the input length. 

Maybe we should be more transparent and convert it to hex directly?
```suggestion
    MASTER_CERTIFICATE_KEY=$(head -c32 /dev/random | hexdump -e '"%x"')
```

Reply:
Changed to `MASTER_CERTIFICATE_KEY=$(hexdump -e '"%x"' /dev/random | head -c64)`, sounds good?

Reply:
yes